src/components/home-screen.vue

```diff
@@ -223,6 +228,7 @@ export default {
   height: 8vh;
```

Comment:
Feels a bit small but given the available space, I believe it is fine. Just mentioning it...

Reply:
Let's follow up with a style change in a separate pull request if it proves to be a nuisance.